### PR TITLE
OCM-9296 | test: automate ids:41822,75256 Create/Delete STS cluster successfully via rosacli

### PR DESCRIPTION
**Pull request by yuwang-RH**
Add OperatorRoleArn info in cluster-detail file for testing.
Automate 41822,75256 Create/Delete STS cluster successfully via rosacli

### CODE REVIEW

**Comment by yuwang-RH** [comment]
logs: https://privatebin.corp.redhat.com/?9f5bf89904ad4e8c#8T7Zm6RLkUKp81QReJEGumMa9W2ABNrkNv48zcr7gziV

File: tests/e2e/test_rosacli_cluster_post.go [hunk 1]
```diff
@@ -461,8 +461,55 @@ var _ = Describe("Delete BYO OIDC cluster testing",
 				Expect(err.Error()).To(ContainSubstring("NoSuchEntity"))
 
 			})
+		It("to verifiy the sts cluster is deleted successfully - [id:75256]",
```
Comment (radtriste) [inline tests/e2e/test_rosacli_cluster_post.go:464]:
```suggestion
		It("to verify the sts cluster is deleted successfully - [id:75256]",
```

File: tests/utils/profilehandler/interface.go [hunk 2]
```diff
@@ -77,13 +77,14 @@ type UserData struct {
...
-	InfraID         string `json:"infra_id,omitempty"`
-	OIDCEndpointURL string `json:"oidc_endpoint_url,omitempty"`
+	APIURL          string   `json:"api_url,omitempty"`
+	ClusterID       string   `json:"cluster_id,omitempty"`
+	ClusterName     string   `json:"cluster_name,omitempty"`
+	ClusterType     string   `json:"cluster_type,omitempty"`
+	ConsoleURL      string   `json:"console_url,omitempty"`
+	InfraID         string   `json:"infra_id,omitempty"`
+	OIDCEndpointURL string   `json:"oidc_endpoint_url,omitempty"`
+	OperatorRoleArn []string `json:"operator_role_arn,omitempty"`
```
Comment (radtriste) [inline tests/utils/profilehandler/interface.go:87]: Please rename to `OperatorRoleArns` so we expect many of them

File: tests/e2e/test_rosacli_cluster_post.go [hunk 3]
```diff
@@ -461,8 +461,55 @@ var _ = Describe("Delete BYO OIDC cluster testing",
...
+				Expect(err).To(BeNil())
+				clusterList, err := clusterService.ReflectClusterList(clusterListout)
+				Expect(err).To(BeNil())
+				Expect(clusterList.IsExist(clusterID)).To(BeFalse())
+
+				By("Check account roles are deleted")
+				var accountRoleArns []string
+				accountRoleArns = append(accountRoleArns, clusterConfig.Aws.Sts.RoleArn)
+				accountRoleArns = append(accountRoleArns, clusterConfig.Aws.Sts.SupportRoleArn)
+				accountRoleArns = append(accountRoleArns, clusterConfig.Aws.Sts.WorkerRoleArn)
```
Comment (radtriste) [inline tests/e2e/test_rosacli_cluster_post.go:497]: Doesn't it work like that ?
```suggestion
				accountRoleArns := []string{
				    clusterConfig.Aws.Sts.RoleArn,
				    clusterConfig.Aws.Sts.SupportRoleArn,
				    clusterConfig.Aws.Sts.WorkerRoleArn,
				}
```

**Review by radtriste — COMMENTED** [review]
/lgtm

**Pull request merged**